app/ante_handler.go

```diff
@@ -41,7 +40,6 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		ante.NewSetUpContextDecorator(),
 		ante.NewExtensionOptionsDecorator(options.ExtensionOptionChecker),
 		// temporarily disabled so that chain can be tested locally without the provider chain running
-		consumerante.NewMsgFilterDecorator(options.ConsumerKeeper),
```

Comment:
Why is this decorator unnecessary?

Reply:
It was originally disabled but enabling it blocked some IBC msgs while running integration tests(redemption).
So I removed it.

```
 ✗ [INTEGRATION-BASIC-GAIA] redemption works
   (from function `WAIT_FOR_STRING' in file dockernet/tests/integration_tests.bats, line 0,
    in test file dockernet/tests/integration_tests.bats, line 210)
     `WAIT_FOR_STRING $STRIDE_LOGS "\[REDEMPTION] completed on $HOST_CHAIN_ID"' failed with status 130
   code: 1
   codespace: undefined
   data: ""
   events: []
   gas_used: "0"
   gas_wanted: "0"
   height: "0"
   info: ""
   logs: []
   raw_log: tx contains unsupported message types at height 353
   timestamp: ""
   tx: null
   txhash: 9783021376368CC3B5AD692940B006446B4E781A1446DD4DCCACACC277539115

   Received SIGINT, aborting ...

   bats warning: Executed 6 instead of expected 9 tests
```